bindata/network/host-network/ns.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    policy-group.network.openshift.io/host-network: ""
```

Comment:
Meh. This makes sense:

```
  - from:
      - namespaceSelector:
          matchLabels:
            network.openshift.io/policy-group: host-network
```

This is weird-looking:

```
  - from:
      - namespaceSelector:
          matchLabels:
            policy-group.network.openshift.io/host-network: ""
```

For other "policy-group" values, there will never be any overlap (ie, you aren't going to have monitoring and routers in the same namespace). Maybe we should just leave the other labels the way they are, and use something non-conflicting for the host-network label? Since it's kind of a hack anyway, maybe use a label that emphasizes its hackiness? Like, `network.openshift.io/policy-group-special: host-network` ?

Other than that, the PR looks good.

Reply:
Does `network.openshift.io/policy-group/host-network: ""` read better?

Reply:
I would definitely prefer a consistent format, so `network.openshift.io/policy-group/host-network` reads better to me.

Reply:
no, it's the "matching an empty string" thing that I don't like. It breaks the ordinary semantics of labels.

Reply:
would you feel better with `network.openshift.io/policy-group/host-network: "true"`?

Reply:
> would you feel better with `network.openshift.io/policy-group/host-network: "true"`?

we don't do that, and that's also annoying for users anyway because the quotes are unexpectedly mandatory (since `true` without quotes would be parsed as a boolean rather than a string)

I guess given the way labels work in k8s, `policy-group.network.openshift.io/host-network: ""` is correct. :slightly_frowning_face:

Reply:
I am going to lean toward: `network.openshift.io/policy-group/host-network: ""`, it reads better and like @squeed said provides a more consistent format.

Reply:
Yes, changed back to the original format @squeed had suggested.